### ISSUE TITLE
Broken link in core/development/policy/test-standard

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4183]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4183|
|Type|defect|
|Created|2009-12-22 01:42:06Z|

link target is "../testing.xhtml"

should be: "../../howto/testing.xhtml"

patch attached

Attachments:

* [testing_standard_bad_link_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e3a/e3a595f0b4d2579156f3a611acb77659b7dac46e/eafa9f289869d50f0eb43b647daba40738aa847e.patch) (492 bytes) - added by khorn on 2009-12-22 01:42:41Z - patch to fix bad link in test-standard.xhtml

<details><summary>Searchable metadata</summary>

```
trac-id__4183 4183
type__defect defect
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1261446126000000 1261446126000000
changetime__1261614688000000 1261614688000000
version__None None
owner__ 
cc__khorn cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn set owner to @khorn|
|-|-|

send to review

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27867]) Apply testing_standard_bad_link_fix.patch: Fix broken link in core/development/policy/test-standard.

Author: khorn
Reviewer: thijs
Fixes: [#4183](https://github.com/twisted/twisted/issues/4183)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
